news/2022-02-07-beatmap-spotlights-season-5-winter-2022.md

```diff
@@ -0,0 +1,167 @@
+
+## Rewards
+
+| Badges | Bracket Tier | Placement |
+| :--: | :--: | :--: |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/ri_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/ri_2.png) | Rhythm Incarnate | The best of the best |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/diamond_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/diamond_2.png) | Diamond | 3% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/platinum_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/platinum_2.png) | Platinum | 3% - 10% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/gold_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/gold_2.png) | Gold | 10% - 25% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/silver_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/silver_2.png) | Silver | 25% - 50% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/bronze_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/bronze_2.png) | Bronze | 50% - 70% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/copper_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/copper_2.png) | Copper | 70% - 95% |
+| ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/iron_1.png) ![](/wiki/Beatmap_Spotlights/img/badges/winter_2022/osu/iron_2.png) | Iron | 95% - 100% |
+
+The percentage values may vary a little depending on the game mode and/or participant count.
+
+The Rhythm Incarnate tier threshold is picked manually based on season's participant count and general size of other tiers, though it is an absolute number that ranges between top 2 and top 50 in most cases.
+
+## <a id="playlists"></a> Spring Season 2021 first playlists
+
+### osu! (lead by [Nowaie](https://osu.ppy.sh/users/5428909))
+
+#### Playlist A (unorthodox rhythm)
+
+- [Polyphia - O.D. \[Hard\]](https://osu.ppy.sh/beatmapsets/1378026#osu/2850639), suggested by [Lefafel](https://osu.ppy.sh/users/2295850)
+- [OSTER project - piano x forte \[Hard\]](https://osu.ppy.sh/beatmapsets/1296249#osu/2983635), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)
+- [Osamu Kubota - Presto \[Jval's Another\]](https://osu.ppy.sh/beatmapsets/1452332#osu/3005290), suggested by [Zelq](https://osu.ppy.sh/users/8953955)
+- [Neru - Lost One no Goukoku \[eveless\]](https://osu.ppy.sh/beatmapsets/83547#osu/230641), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
+- [OSTER project - Violet Rose \[Insane\]](https://osu.ppy.sh/beatmapsets/1178488#osu/2457698), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)
+- [onoken - ZADAMGA \[Luscent's Extra\]](https://osu.ppy.sh/beatmapsets/1492550#osu/3188917), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)

```

Comment:
As mentioned by Lefafel internally, it would be nice to mention them too as the map was both nominated by them and DigitalHypno